packages/core/src/utils/objects/methodsOf.spec.ts

```
@@ -0,0 +1,34 @@
+import {isSymbol, isSymbolOrSymbolClass} from "./isSymbol.js";
+import {methodsOf} from "./methodsOf.js";
+import {Inject} from "@tsed/di";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)